tslint.json

```diff
@@ -0,0 +1,37 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+      "tslint:recommended",
+      "tslint-react",
+      "tslint-eslint-rules",
+      "tslint-config-prettier"
+    ],
+    "linterOptions": {
+      "exclude": [
+          "node_modules/**"
+      ]
+    },
+    "jsRules": {},
+    "rules": {
+      "interface-name": [true, "never-prefix"],
+      "no-console": false,
+      "max-classes-per-file": false,
+      "no-any": false,
+      "no-empty": false,
+      "prefer-for-of": false,
+      "jsx-no-lambda": [false],
+      "no-empty-interface": [false],
+      "object-literal-sort-keys": false,
```

Comment:
Can we please disable `no-empty-interface` and `object-literal-sort-keys`.

Reply:
sorry not sure if I understand, they are already set to false

Reply:
Oh, sorry, ignore then.